.devcontainer/Dockerfile

```diff
@@ -0,0 +1,38 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+ARG IMAGE_TYPE
+FROM tensorflow/tensorflow:devel$IMAGE_TYPE
+
+# This Dockerfile adds a non-root user with sudo access. Update the “remoteUser” property in
+# devcontainer.json to use it. More info: https://aka.ms/vscode-remote/containers/non-root-user.
+ARG USERNAME=vscode
```

Comment:
We could remove this with another PR in the next months when our `user` will be ready by default: https://github.com/tensorflow/build/pull/21#issuecomment-805776018